### PR TITLE
doc: add some common errnos to iochaos document

### LINKED ISSUE #136: The IO doc is not very correct
## Bug Report

**What version of Kubernetes are you using?**

None

**What did you do?**

None

**What did you expect to see?**

- What is the common error number for IO injection?
- What happend when set errno empty and meet other errno such as `EL2HLT 51`
- The errno `32` is broken pipe,Seems not a IO error
- The [Linux system error](https://www-numi.fnal.gov/offline_software/srt_public_context/WebDocs/Errors/unix_system_errors.html) create date is 2004, should we update a new link?

**What did you see instead?**
The error number in [create a yaml](https://github.com/pingcap/chaos-mesh/blob/master/doc/io_chaos.md#define-the-chaos-yaml-file) is misunderstand. The "32" is not a IO error number

### PR DESCRIPTION
**Pull request by cwen0**
Signed-off-by: cwen0 <cwenyin0@gmail.com>  
### What problem does this PR solve? 
fix #136 
### What is changed and how does it work?
add some common errnos to iochaos document

### CODE REVIEW

File: doc/io_chaos.md [hunk 1]
```diff
@@ -96,10 +96,13 @@ Description:
 * **duration**: represents the duration of a chaos action. The duration might be a string with the signed sequence of decimal numbers, each with optional fraction and a unit suffix, such as `"300ms"`, `"-1.5h"` or `"2h45m"`.
 * **delay**: defines the value of IO chaos action delay. The duration might be a string with the signed sequence of decimal numbers, each with optional fraction and a unit suffix, such as `"300ms"`, `"-1.5h"` or `”2h45m”`. Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", and "h".
   If `Delay` is empty, the operator will generate a value for it randomly.
-* **errno**: defines the error code that is returned by an IO action. It is an int32 string like `"32"`. This field need to be set when you choose an `errno` or `mixed` action. If `errno` is empty, the operator will randomly generate an error code for it. You can set the `errno` by referring to [Errors: Linux System Errors](https://www-numi.fnal.gov/offline_software/srt_public_context/WebDocs/Errors/unix_system_errors.html).
```
[the remark this answers is not in the document]
Comment (cwen0) [reply doc/io_chaos.md]: the random erron was generated by the fuse server

[the remark this answers is not in the document]
Comment (zhouqiang-cl) [reply doc/io_chaos.md]: so，if the error code is 32，what happend

[the remark this answers is not in the document]
Comment (cwen0) [reply doc/io_chaos.md]: just return a this error to the application

[the remark this answers is not in the document]
Comment (zhouqiang-cl) [reply doc/io_chaos.md]: lgtm

**Pull request merged**